### PR TITLE
Pms 3827

### PR DESCRIPTION
**Pull request by Fimmer**
Replace rqr_code_png gem with rqr_code gem.
Reworking with the structure of the rqr_code gems caused CASino to fail when trying to create pngs of QRcodes. Replaced pngs with svgs.

### CODE REVIEW

**Review by MikhailFedotovPeytzDK — APPROVED** [review]

**Comment by MikhailFedotovPeytzDK** [comment]
Well, can't find any obvious flaw :)

File: app/helpers/casino/two_factor_authenticators_helper.rb [hunk 1]
```diff
@@ -1,25 +1,16 @@
...
-  def otp_qr_code_suggested_size(data)
-    data_bits = data.length * 8
-    (3..40).each do |size|
-      metadata_bits = 4 + RQRCode::QRUtil.get_length_in_bits(RQRCode::QRMODE[:mode_8bit_byte], size)
-      total_data_bits = metadata_bits + data_bits
-      max_data_bits = RQRCode::QRCode.count_max_data_bits(RQRCode::QRRSBlock.get_rs_blocks(size, 1))
-      return size if total_data_bits < max_data_bits
-    end
+    qr = RQRCode::QRCode.new(auth_url)
+    svg = qr.as_svg(
```
Comment (MikhailFedotovPeytzDK) [inline app/helpers/casino/two_factor_authenticators_helper.rb]: A bit of nitpicking :) we don't use this `svg` variable, do we? So we may just make it a bit shorter, like `RQRCode::QRCode.new(auth_url).as_svg(...).html_safe`?

**Comment by MikhailFedotovPeytzDK** [comment]
BTW I had to merge this one into https://github.com/Peytz/CASino/pull/2 otherwise the upgrade to 2.7.7 / Rails 5 was failing with exactly same problem :)

File: app/helpers/casino/two_factor_authenticators_helper.rb (hunk 1 above)
Comment (Fimmer) [reply app/helpers/casino/two_factor_authenticators_helper.rb]: You are right. This is me not cleaning up after myself. Was testing out returning svg or png or both for no obvious reason :)

Comment (Fimmer) [reply app/helpers/casino/two_factor_authenticators_helper.rb]: @MikhailFedotovPeytzDK made the change you suggested

**Pull request merged**